src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
@graywolf336 I do not like that at all, honestly.
> all development will be done in TypeScript, as the “principal architect” behind this I will refuse to budge.
I am sure that you are even more disappointed by that decision. I cannot truly understand the worries. 
Maybe, pure JS-developers will not feel familiar with Typescript (but then, you can also get rid of classes in this step as well ;) ). On the other hand, there's a huge community of developers coming from Java or any other typed language environment which don't feel familiar with JS but are happy to use TypeScript.
At the same time, I see the need of making adoption as easy as possible.